survey/survey-service-offering_instance.json

```diff
@@ -0,0 +1,123 @@
+
+{
+       "@context":{
+          "xsd":"http://www.w3.org/2001/XMLSchema#",
+          "survey":"https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/survey/",
+          "gx": "https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#",
+          "general": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/"
+        },
+       "@id":"did:example:survey_service_offering_0xy5vhw3heyjqb4d347hc9zrkw",
+       "@type":"survey:SurveyServiceOffering",
+       "survey:surveyUrl":{
+          "@value":"undefined?session=0xy5vhw3heyjqb4d347hc9zrkw",
+          "@type":"xsd:string"
+       },
+       "survey:surveyCreationTime":{
+          "@value":"2023-08-01T15:14:04",
+          "@type":"xsd:dateTime"
+       },
+       "survey:surveyEndTime":{
+          "@value":"2023-08-02T17:14:00",
+          "@type":"xsd:dateTime"
+       },
+       "gx:description":{
+          "@value":"asdwescxvf",
+          "@type":"xsd:string"
+       },
+       "gx:name":{
+          "@value":"asdfdfdfeacxx",
+          "@type":"xsd:string"
+       },
+       "survey:general":{
+         "@id": "did:web:registry.gaia-x.eu:General:PlFjHxuQoH0v7iaVC0QqdxmtQyqY-LQX6D-p",
+         "@type": "general:General",
+         "general:description": {
+           "@type": "general:Description",
+           "gx:name": {
+             "@value": "3D Model in Grafing",
+             "@type": "xsd:string"
+           },
+           "gx:description": {
+             "@value": "town district, with traffic signs",
+             "@type": "xsd:string"
+           }
+         },
+         "general:data": {
+           "@type": "general:Data",
+           "general:size": {
+             "@value": "5.8",
+             "@type": "xsd:float"
+           },
+           "general:contractId": {
+             "@value": "contract_zcdkr7kqd47y0w5b4tg91w1etw",
+             "@type": "xsd:string"
+           },
+           "general:recordingTime": {
+             "@value": "2024-04-01T00:00:00",
+             "@type": "xsd:dateTime"
+           }
+         },
+         "general:links": {
+           "@type": "general:Links",
+           "general:data": {
+             "@type": "general:Link",
+             "general:url": {
+               "@value": "https://example.com/video.mpg",
+               "@type": "xsd:anyURI"
+             },
+             "general:type": "Video"
+           },
+           "general:media": {
```

Comment:
you can leave this field out if it is not necessary for a survey (it is not mandatory). same for the other json file.